wiki/Game_modes/en.md

```diff
@@ -0,0 +1,50 @@
+<!-- TODO: 
+- retake images to be ASC-compliant (update with respective game mode stubs)
+- add image for selecting game mode through Mode button -->
+
+# Game modes
+
+In osu!, a **game mode** is one of the four distinct configurations included in the osu! client (osu!, osu!taiko, osu!catch, and osu!mania). All of which are based on, or inspired by, other existing rhythm video games. 
```

Comment:
```suggestion
In osu!, a **game mode** is one of the four distinct gameplay configurations included in the osu! client (osu! (game mode), osu!taiko, osu!catch, and osu!mania). All of which are based on, or inspired by, other existing rhythm video games. 
```

Reply:
replaced the added parantheses with brackets for correct syntax; will unresolve if you disagree.